sr_port/hd.mpt

```diff
@@ -11,13 +14,17 @@
 %HD	;GT.M %HD utility - hexadecimal to decimal conversion program
 	;invoke at INT with %HD in hexadecimal to return %HD in decimal
 	;invoke at FUNC as an extrinsic function
+	;Note: works even if input hex number contains 0x or 0X prefix
 	;if you make heavy use of this routine, consider $ZCALL
 	;
 	s %HD=$$FUNC(%HD)
 	q
 INT	r !,"Hexidecimal: ",%HD s %HD=$$FUNC(%HD)
 	q
 FUNC(h)
+	n prefix
+	set prefix=$zextract(h,1,2)
+	set:(("0X"=prefix)!("0x"=prefix)) h=($zextract(h,3,99))
 	q:$tr(h,"E","e")<0 ""
```

Comment:
This is most likely to avoid NUMOFLOW errors. See example below E vs e.

```
YDB>s h="1E98" 
YDB>w h<0
%GTM-E-NUMOFLOW, Numeric overflow

YDB>s h="1e98"
YDB>w h<0
0

```

Reply:
OK